core-developers/become-core-developer.rst

```diff
@@ -59,6 +60,41 @@ are granted through these steps:
      were in the form of a separate post on the already open topic with
      the poll.
 
+Poll template
+=============
+
+.. _coredev-template:
+
+While Discourse uses Markdown for formatting, the poll functionality is
+custom and somewhat resembles BBcode. There's a creator for polls in the
+UI (click the Cog icon in the edit box toolbar and choose "Build Poll").
+Make sure to choose "Show Results... When the poll is closed".
+
+Alternatively, here is a template for a poll that you can copy:
+
+.. code-block:: bbcode
+
+   [poll type=regular results=on_close public=false chartType=bar groups=committers close=2024-07-15T21:15:00.000Z]
+   * Promote Basil Fawlty
+   * Do not promote
+   [/poll]
+
+.. raw:: html
+
+    <script>
```

Comment:
What’s this for?

Reply:
It updates the close date in the template to now+7 days.

See the preview: https://cpython-devguide--1396.org.readthedocs.build/core-developers/become-core-developer/#poll-template

Reply:
I would prefer to remove the Javascript. I think that anyone can pick a date to close a vote in one week.

Reply:
Sometimes easier said than done, but we have had polls that needed resetting a couple of times due to timezone confusion.